charts/external-dns/values.yaml

```
@@ -131,7 +131,12 @@ extraArgs: []
 secretConfiguration:
   enabled: false
   mountPath: ""
+  subPath: ~
```

```suggestion
  subPath: ""
```
The default should be empty.